tests/learning_nuggets/test_models.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+from django.urls import reverse
+from wagtail.models import Page
+
+from apps.learning_nuggets.models import LearningCategory
+from apps.learning_nuggets.models import LearningCenterPage
+from apps.learning_nuggets.models import LearningNuggetPage
+
+
+@pytest.mark.django_db
+def test_learning_center_page_is_single_instance():
+    main_page = LearningCenterPage.objects.create
+
+    assert LearningCenterPage.objects.first() == main_page
+    assert LearningCenterPage.objects.count() == 1
```

Comment:
I added the assertions in this test, otherwise the lint didn't let me commit. I hope it was that you had in mind for this testing case.